e2e/cypress/integration/messaging/collapse_link_spec.js

```diff
@@ -0,0 +1,126 @@
+// - [*] indicates an assertion (e.g. * Check the title)
+// - Use element ID when selecting an element. Create one if none.
+// ***************************************************************
+
+describe('Link preview - Removing it from my view removes it from other user\'s view', () => {
+    before(() => {
+        // # Set the configuration on Link Previews
+        setSystemConsoleLinkPreview();
+        setAccountSettingsLinkPreview('user-1');
+        setAccountSettingsLinkPreview('sysadmin');
+
+        // # Login and go to /
+        cy.apiLogin('user-1');
+        cy.visit('/');
+    });
+
+    it('M18708-Link preview - Removing it from my view removes it from other user\'s view', () => {
+        const message = 'https://www.bbc.com/news/uk-wales-45142614';
+
+        // # Create new DM channel with user's email
+        cy.apiGetUsers(['user-1', 'sysadmin']).then((userResponse) => {
+            const userEmailArray = [userResponse.body[1].id, userResponse.body[0].id];
+
+            cy.apiCreateDirectChannel(userEmailArray).then(() => {
+                cy.visit('/ad-1/messages/@sysadmin');
+
+                // # Post message to use
+                cy.postMessage(message);
+
+                cy.getLastPostId().then((postId) => {
+                    // * Check the preview is shown
+                    cy.get(`#${postId}_message`).find('.file-preview__button').should('exist');
+
+                    // # Log-in to the other user
+                    cy.apiLogin('sysadmin');
+                    cy.visit('/ad-1/messages/@user-1');
+
+                    // * Check the preview is shown
+                    cy.get(`#${postId}_message`).find('.file-preview__button').should('exist');
+
+                    // # Log-in back to the first user
+                    cy.apiLogin('user-1');
+                    cy.visit('/ad-1/messages/@sysadmin');
+
+                    // # Collapse the preview
+                    cy.get(`#${postId}_message`).find('.post__embed-visibility').click({force: true});
+
+                    // * Check the preview is not shown
+                    cy.get(`#${postId}_message`).find('.file-preview__button').should('not.exist');
+
+                    // # Log-in to the other user
+                    cy.apiLogin('sysadmin');
+                    cy.visit('/ad-1/messages/@user-1');
+
+                    // * Check the preview is shown
+                    cy.get(`#${postId}_message`).find('.file-preview__button').should('exist');
+
+                    // # Log-in back to the first user
+                    cy.apiLogin('user-1');
+                    cy.visit('/ad-1/messages/@sysadmin');
+
+                    // # Expand one more time the preview
+                    cy.get(`#${postId}_message`).find('.post__embed-visibility').click({force: true});
+
+                    // # Remove the preview
+                    cy.get(`#${postId}_message`).find('#removePreviewButton').click({force: true});
```

Comment:
This failed because there's no element ID of `removePreviewButton`.  I'd suggest to add a new one but as `data-testid='removeLinkPreviewButton'` [here](https://github.com/larkox/mattermost-webapp/blob/CypressCollapseLink/components/post_view/post_attachment_opengraph/post_attachment_opengraph.jsx#L207). Then it could be written as,
```javascript
cy.get(`#${postId}_message`).within(() => {
    cy.findByTestId('removeLinkPreviewButton').click({force: true});
});
```

Reply:
This id used to exist on my previous local version, but it was removed on master branch. I will add it and use it.